plugins/hls-eval-plugin/src/Ide/Plugin/Eval/CodeLens.hs

```diff
@@ -295,8 +285,7 @@ runEvalCmd plId st EvalParams{..} =
                 dbg "LOAD RESULT" $ asS loadResult
                 case loadResult of
                     Failed -> liftIO $ do
-                        hClose logHandle
-                        err <- readFile logFilename
+                        let err = ""
```

Comment:
suspicious

Reply:
Yes it is. Previously the `err` was filled by the logging architecture (see discussion here: https://github.com/haskell/haskell-language-server/pull/2669#discussion_r801201024) and @pepeiborra agreed that it can be removed. We can pull the thread even more by removing this `err` value, changing the return type from `Either` to `Maybe`. I tried to reduce the amount of changes in order to restore this plugin.

Reply:
If it's something to do in a follow-up PR, please leave a comment. Otherwise future readers (possibly future you :) ) will be mystified.

Reply:
I think the simplest thing to do is fill the `err` with `asS loadResult`, or something even simpler like "failed to load module". I feel like it will be hard to change the `Either` to a `Maybe` because `type CommandFunction idestate a = ideState a -> LspM Conflg (Either ResponseError Value)`, unless we're changing `CommandFunction` which would change lots of things should just put some reason in there instead.

Reply:
Thank you, I'll address that in a followup commit.